gbus/bus.go

```diff
@@ -549,10 +587,10 @@ func (b *DefaultBus) monitorAMQPErrors() {
 	}
 }
 
-func (b *DefaultBus) sendImpl(ctx context.Context, tx *sql.Tx, toService, replyTo, exchange, topic string, message *BusMessage, policies ...MessagePolicy) (er error) {
+func (b *DefaultBus) sendImpl(sctx context.Context, tx *sql.Tx, toService, replyTo, exchange, topic string, message *BusMessage, policies ...MessagePolicy) (er error) {
 	b.SenderLock.Lock()
 	defer b.SenderLock.Unlock()
-	span, ctx := opentracing.StartSpanFromContext(ctx, "sendImpl")
+	span, _ := opentracing.StartSpanFromContext(sctx, "sendImpl")
```

Comment:
shouldn't the wrapped context (the one that is returned from the StartSpanFromContext method) be used ?

Reply:
We don't use it anywhere down the road...